src/grading.asm

```diff
@@ -213,24 +222,29 @@ SECTION "Grading Functions", ROM0
 GradeInit::
     xor a, a
     ld [wDecayRate], a
-    ld [wGradePoints], a
+    ld [wInternalGradePoints], a
     ld [wInternalGrade], a
     ld [wDisplayedGrade], a
     ld [wRankingDisqualified], a
     ld [wEffectTimer], a
     ld [wDecayCounter], a
     ld [wGradeGauge], a
+    ld [wSubgrade], a
 
     ; Most modes begin ungraded.
     ld a, GRADE_NONE
     ld [wDisplayedGrade], a
 
-    ; TGM1 and DMGT are the exceptions.
+    ; TGM1, TGM3, CHIL and DMGT are the exceptions.
     ld a, [wSpeedCurveState]
     cp a, SCURVE_TGM1
     jr z, .grade9start
     cp a, SCURVE_DMGT
     jr z, .grade9start
+    cp a, SCURVE_TGM3
+    jr z, .grade9start
+    cp a, SCURVE_CHIL

```

Comment:
Not sure if I'm personally a fan of having grades in CHIL mode at all. It's supposed to be a chill mode after all.

Reply:
well, it's chill, an easy-to-get GM